python/sglang/srt/layers/moe/topk.py

```diff
@@ -204,6 +203,16 @@ def is_power_of_two(n):
     return n > 0 and math.log2(n).is_integer()
 
 
+# TODO will fuse this into kernel, thus use slow manual operation now
+@torch.compile(dynamic=True, backend=get_compiler_backend())
+def _mask_topk_ids_padded_region(
+    topk_ids: torch.Tensor,
+    num_token_non_padded: Optional[torch.Tensor] = None,
```

Comment:
The code does not allow `num_token_non_padded` to be None.

Reply:
In addition, is it necessary to wrap up `num_token_non_padded` as a Tensor? It if is an integer, we can directly call `topk_ids[num_token_non_padded:] = -1`

Reply:
the second one does not seem to be feasible IMHO